holidays/countries/south_georgia_and_the_south_sandwich_islands.py

```
@@ -0,0 +1,152 @@
+
+    country = "GS"
+    default_language = "en_GS"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    supported_languages = ("en_GS", "en_US")
+    start_year = 2012
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, SouthGeorgiaandtheSouthSandwichIslandsStaticHolidays)
```

Ruff (0.12.2)

Undefined name `SouthGeorgiaandtheSouthSandwichIslandsStaticHolidays`

(F821)